tests/helpers.ts

```diff
@@ -23,6 +23,10 @@ import * as path from 'path';
 import * as balenaCLI from '../build/app';
 import { configureBluebird } from '../build/app-common';
 
+// REMOVE When setMaxListeners merged
```

Comment:
This meant PR #1529, right? I've finally merged it, which also means this PR now needs rebasing - sorry! :-)